### PR TITLE
Base structure

### PR DESCRIPTION
**Pull request by JhanviAlbiorix**

### CODE REVIEW

File: lib/src/presentation/core/utils/helper/shared_preference_helper.dart
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/helper/shared_preference_helper.dart:1]: empty file exists

File: lib/src/presentation/core/utils/helper/validators.dart
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/helper/validators.dart:1]: commented file exists

File: lib/src/presentation/core/utils/widgets/custom_text_form_field.dart
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/widgets/custom_text_form_field.dart:1]: empty file exists

File: pubspec.lock
Comment (SohanAlbiorix) [inline pubspec.lock:1]: .lock file should be not in git repo, please remove this and add it to .gitignore

File: web/index.html
Comment (SohanAlbiorix) [inline web/index.html:1]: web index file? Have we added support for the web?

File: lib/src/presentation/core/utils/widgets/custom_button.dart
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/widgets/custom_button.dart:1]: I can see most of the properties are passing, so it doesn't make sense to create a custom button.

**Review by SohanAlbiorix — CHANGES_REQUESTED** [review]

File: lib/src/presentation/core/utils/helper/base_validations_strings.dart
Comment (SohanAlbiorix) [inline lib/src/presentation/core/utils/helper/base_validations_strings.dart:1]: I believe your app supports localisation, please update this file once you implemented it. It is okay for now.

Comment (JhanviAlbiorix) [reply lib/src/presentation/core/utils/helper/base_validations_strings.dart:1]: I have made the initial commit considering the structure as functionalities are asked to be ignored for now as per yesterday's discussion.

File: pubspec.lock
Comment (SohanAlbiorix) [reply pubspec.lock:1]: This file should be removed from the version control, you have already added it to gitignore.

**Review by SohanAlbiorix — APPROVED** [review]

**Pull request merged**